packages/maskbook/src/plugins/Trader/pipes.ts

```diff
@@ -108,6 +125,8 @@ export function resolveTradePairLink(tradeProvider: TradeProvider, address: stri
             return `https://info.uniswap.org/pair/${address}`
         case TradeProvider.ZRX:
             return ''
+        case TradeProvider.ONE_INCH:
+            return ''
```

Comment:
Can we use the pool link here?

https://app.1inch.io/#/1/dao/pools/0x6a11f3e5a01d129e566d783a7b6e8862bfd66cca/governance